pkg/curatedpackages/curatedpackages.go

```diff
@@ -153,3 +154,25 @@ func GetRegistry(uri string) string {
 	}
 	return uri[:lastInd]
 }
+
+func UpgradePackages(
+	ctx context.Context,
+	kubeConfig string,
+	kubectl *executables.Kubectl,
+	source BundleSource,
+	targetBundleVersion string,
+) error {
+	b := NewBundleReader(
+		kubeConfig,
+		source,
+		kubectl,
+		nil,
+		nil,
+	)
+	activeController, err := b.GetActiveController(ctx)
+	if err != nil {
+		return err
+	}
+	err = b.UpgradeBundle(ctx, activeController, targetBundleVersion)
+	return err
```

Comment:
```suggestion
	return b.UpgradeBundle(ctx, activeController, targetBundleVersion)
```

Reply:
This commit only moves this code from https://github.com/aws/eks-anywhere/pull/2863/files/069fb7b7afe5364851ce8e91bcca8f3792a335c9#diff-55361c4daff82b0371cefa04eada4c480ef3b0a92e8cc93edd7dcbeee37438cfL49
I'd use another PR for this as this is not related to metallb E2E.

Reply:
I actually had removed some redundancy already but forgot some after moving the code.